credsweeper/scanner/scan_type/pem_key_pattern.py

```diff
@@ -58,15 +60,12 @@ def is_pem_key(cls, lines: List[str], config: Config) -> bool:
         lines = cls.strip_lines(lines)
         lines = cls.remove_leading_config_lines(lines)
```

Comment:
About checking line length is over `190` or not, this method `remove_leading_config_lines()` can reduce size of `lines`.
So isn't it good to just remain line length check code here?

Like below.